README.md

```diff
@@ -25,3 +28,41 @@ Look at the CSS for the individual components for usage examples and notes.
 - [Related Items](source/stylesheets/modules/_related-items.scss)
 - [Skip Link](source/stylesheets/modules/_skip-link.scss)
 - [Sub Navigation](source/stylesheets/modules/_sub-navigation.scss)
+
+
+## Usage
+
+For now, please copy the SASS and Javsacript for the components that you require
```

Comment:
Alright, this is the most pedantic thing ever, but can we make this [Sass, not SASS](http://sassnotsass.com/)?